docs/source/notes/windows.rst

```diff
@@ -165,13 +165,18 @@ Import error
 
 
 The problem is caused by the missing of the essential files. Actually,
-we include almost all the essential files that PyTorch need except VC2017
-redistributable. You can resolve this by typing the following command.
+we include almost all the essential files that PyTorch need for the conda
+package except VC2017 redistributable. 
+You can resolve this by typing the following command.
 
 .. code-block:: bat
 
     conda install -c peterjc123 vc vs2017_runtime
 
+As for the wheels package, since we didn't pack the intel-openmp files in,
+please make sure you install Numpy+MKL from this `site
+<https://www.lfd.uci.edu/~gohlke/pythonlibs/#numpy>`_ instead of Numpy.
```

Comment:
This is a pretty random link. Is there really no better resource for that?

Reply:
But we don't need scipy, right? I'd rather use the PyPI package if this is enough to resolve the problems.